frontend/src/Review.js

```diff
@@ -42,7 +42,7 @@ class ReviewContainer extends React.Component {
     }
 
     sendData = (rest_id, review) => {
-        fetch("/submit_review", {
+        fetch("/cravr/submit_review", {
```

Comment:
I see another fetch("<function in app.py>") on line 12 of this file. Should that be modified in this way as well?

Reply:
Yes, youre very correct. I missed that when I did a full directory search for fetch somehow...